magicctapipe/scripts/lst1_magic/semi_automatic_scripts/clusters.py

```diff
@@ -20,6 +20,8 @@ def slurm_lines(queue, job_name, array=None, mem=None, out_name=None):
         Requested memory. If None cluster default (5 GB) will be used
     out_name : None or str
         If the output should be written to a specific output file
+    nice_parameter : None or int
```

Comment:
Again, maybe there just int (or you should add a line of code later that do something like: 'ok if this is none put it to 0' (but easier to ask only for an int)

Reply:
the idea was somewhat different: if it is specified as None this line is not added to the output